doc/api/esm.md

```diff
@@ -456,39 +454,38 @@ package might look like this:
   "type": "module",
   "main": "./pkg.cjs",
   "exports": {
-    "require": "./pkg.cjs",
-    "default": "./pkg.mjs"
+    ".": "./pkg.cjs",
+    "./module": "./pkg.mjs"
```

Comment:
I really don't like the idea of advising a difference best-practice now, just because our current solution has been agreed to be flagged until January.

This approach can be mentioned (and I believe it is?), but we should document our best consensus (as is documented currently) as to what we think users should be doing, even though we still have the feature flagged, as users deserve to know where things will be going.

Yes, in theory, things can change again here - but then we would be updating the docs to the new best approach.

Reply:
This particular section isn't advising a best practice, it's introducing the hazard. All the sections suggesting best practices recommend using conditional exports.

Reply:
See my comment above.